lib/cloud_controller/runner.rb

```diff
@@ -29,6 +29,11 @@ def initialize(argv)
       secrets_hash = parse_secrets
       parse_config(secrets_hash)
 
+      # DB connection metrics have a label to determine whether the process accessing the connection is the
+      # main or a worker process. We need to set this env variable before `setup_db` otherwise the main process
+      # will show up twice in the metrics as main and worker. Thin metrics will be labeled with main as well.
+      ENV['IS_MAIN'] = 'true'
```

Comment:
**nit** I wonder if something like `ENV['PROCESS_TYPE'] = 'main'` and then set it to `worker` in the puma runner would be more expressive?